Modules/posixmodule.c

```diff
@@ -5470,32 +5470,54 @@ os__path_islink_impl(PyObject *module, PyObject *path)
 /*[clinic input]
 os._path_normpath
 
-    path: object
+    path: unicode
+    /
```

Comment:
I think you should leave the `_path_normpath` changes out of this PR; it is good practice to keep changes focused; avoid PR scope creep. If needed, this can be done in a follow-up PR.

Reply:
I reverted it, but I thought it was fine since I was already updating that function.